tests/gcp/utils/base_gcp_system_test_case.py

```diff
@@ -79,20 +79,17 @@ def retrieve_variables():
 SKIP_TEST_WARNING = """
```

Comment:
I understand that this class will be gone when you finish the POC?

Reply:
Yes, before removing it all system test will have to be refactored.